src/instana/instrumentation/redis.py

```diff
@@ -2,94 +2,115 @@
 # (c) Copyright Instana Inc. 2018
 
 
+from typing import Any, Callable, Dict, Tuple
 import wrapt
 
-from ..log import logger
-from ..util.traceutils import get_tracer_tuple, tracing_is_off
-
+from instana.log import logger
+from instana.span.span import InstanaSpan
+from instana.util.traceutils import get_tracer_tuple, tracing_is_off
 
 try:
     import redis
 
     EXCLUDED_PARENT_SPANS = ["redis", "celery-client", "celery-worker"]
 
-    def collect_tags(span, instance, args, kwargs):
+    def collect_tags(
```

Comment:
You can change it to below to be consistent with OTel naming conventions.

```suggestion
    def collect_attributes(
```